### PR TITLE
Fixed dist file to match new validation of locales

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/sulu-io/sulu/pull/2175
| License | MIT
| Documentation PR | none

### CODE REVIEW

File: app/Resources/webspaces/sulu.io.xml.dist [hunk 1]
```diff
@@ -86,6 +76,7 @@
                     </urls>
                     <custom-urls>
                         <custom-url>*.sulu.lo</custom-url>
```
Comment (danrot) [inline app/Resources/webspaces/sulu.io.xml.dist]: I would remove that line, since this is not a typical setup

**Comment by danrot** [comment]
I would go even a step further with the localizations in the webspace configuration, and remove them completely in the portal. So it would be the easiest setup possible.

File: app/Resources/webspaces/sulu.io.xml.dist (hunk 1 above)
Comment (chirimoya) [inline app/Resources/webspaces/sulu.io.xml.dist]: There is no typical setup, I guess. Both are valid use case and I would keep this two examples.

**Comment by chirimoya** [comment]
Also:
- simplify URLs configuration, prod at least
- remove resource-locator if possible

**Pull request merged**